torch/functional.py

```diff
@@ -1235,16 +1235,29 @@ def norm(input, p="fro", dim=None, keepdim=False, out=None, dtype=None):  # noqa
             None   Frobenius norm                2-norm
             'fro'  Frobenius norm                --
             'nuc'  nuclear norm                  --
-            Other  as vec norm when dim is None  sum(abs(x)**ord)**(1./ord)
+            Other  --                            sum(abs(x)**ord)**(1./ord)
```

Comment:
Should "Other" be "Number"?

Reply:
Yeah I'll change it. Also, `p=None` is not supported :package: 

```
>>> a.norm(p=None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/endoplasm/miniconda3/lib/python3.7/site-packages/torch/tensor.py", line 339, in norm
    return torch.norm(self, p, dim, keepdim, dtype=dtype)
  File "/home/endoplasm/miniconda3/lib/python3.7/site-packages/torch/functional.py", line 747, in norm
    return torch._C._VariableFunctions.norm(input, p)
TypeError: norm(): argument 'p' (position 2) must be Number, not NoneType
```